sklearnex/spmd/covariance/incremental_covariance.py

```diff
@@ -0,0 +1,27 @@
+# ==============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from onedal.spmd.covariance import (
+    IncrementalEmpiricalCovariance as onedal_IncrementalEmpiricalCovariance,
+)
+
+from ...covariance import (
+    IncrementalEmpiricalCovariance as IncrementalEmpiricalCovariance_batch,
+)
+
+
+class IncrementalEmpiricalCovariance(IncrementalEmpiricalCovariance_batch):
+    _onedal_incremental_covariance = staticmethod(onedal_IncrementalEmpiricalCovariance)
```

Comment:
What does this do?

Reply:
it changes _onedal_incremental_covariance to spmd version